spec/Repository/UiLocationServiceSpec.php

```diff
@@ -108,6 +108,68 @@ function it_deletes_locations(LocationService $locationService)
         $this->deleteLocations([$deleteLocationId]);
     }
 
+    function it_trashes_a_single_location_and_returns_the_parent(LocationService $locationService)
```

Comment:
@bdunogier I did consider it. It was in the controller when I first wrote the code. But I ended up injecting the location service and I thought it could be nice to have to have smaller controller. The service makes this code reusable and testable.
Your comment made me realize the method was poorly named, I improved it, tell me what you think :)